rocky/katalogus/client.py

```diff
@@ -73,6 +73,15 @@ def __str__(self):
         return str(self.message)
 
 
+class DuplicateNameError(KATalogusError):
+    def __init__(self, *args: object, status_code: str | None = None) -> None:
```

Comment:
For this higher level exception, this `status_code` is really not relevant for the end user? Personally I would leave this out and just use the error message as suggested below.